.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["2.7", "3.6", "3.7"]
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
There's also a v2 checkout action. Any reason to stick to v1?

Reply:
no, lazyness, but I added a dependabot config, so it would have fixed that for us soon :)